src/XRP/issued-currency-client.ts

```diff
@@ -192,6 +202,33 @@ export default class IssuedCurrencyClient {
     }
   }
 
+  /**
+   * Subscribes to an account's incoming transactions, and triggers a callback upon recieving each transaction.
```

Comment:
```suggestion
   * Subscribes to all transactions that affect the specified account, and triggers a callback upon receiving each transaction.
```

Reply:
Correct me if I'm wrong, but from what I understood in the docs and I think we discussed, there will be an event for any transaction that affects this account, including an outgoing transaction that modifies its balance.

Reply:
Ah yeah missed that when merging those changes in